src/features/import/utils/problems/predictProblems.ts

```diff
@@ -75,6 +63,7 @@ export function predictProblems(
   customFields.forEach((field) => {
     customFieldsBySlug[field.slug] = field;
   });
+  let isMissingValue = false;
```

Comment:
I am not really sure this variable is doing so much 😅 If I remove it from the code, the only difference is that I get both unconfigured ID error and "not all rows have identifiers" error, instead of just the latter.

Reply:
This is just for removing unconfigured ID error(yellow box). Thought that it is weird to show yellow box with text "I understand and want to proceed anyway". Should I remove it?

Reply:
I think it makes sense to do what you've tried to do @kaulfield23, but I think the best way to do it is to write a test that reproduces the problem you found, and then fix it.

Reply:
@richardolsson Okay! will do 👍

Reply:
I think that case is already covered with the test I wrote ( test : returns problem when either the first name or last name value is missing ) . And the case that ziggi found already covered with another test that already exists ( test  : returns problem when ID or name are not configured )

Is there a case that I have not added? @richardolsson